test/fuzzers/frame_fuzzer.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+#include <folly/String.h>
+#include <folly/io/async/EventBaseManager.h>
+#include <iostream>
+
+#include <folly/init/Init.h>
+#include <folly/portability/GFlags.h>
+#include <glog/logging.h>
+
+#include "rsocket/ConnectionAcceptor.h"
+#include "rsocket/DuplexConnection.h"
+#include "rsocket/RSocketServer.h"
+
+struct FuzzerConnectionAcceptor : public rsocket::ConnectionAcceptor {
```

Comment:
If you're using `struct`, then you don't need the `public` keyword for the super class.  Here and all following classes.